kaspresso/src/main/kotlin/com/kaspersky/kaspresso/testcases/api/testcase/DocLocScreenshotTestCase.kt

```diff
@@ -146,12 +146,12 @@ abstract class DocLocScreenshotTestCase(
                 resourcesDirsProvider,
                 resourceFileNamesProvider
             ),
-            screenshotMaker = CombinedScreenshotMaker(
-                preferredScreenshotMaker = InternalScreenshotMaker(kaspresso.device.activities, screenshotParams),
-                fallbackScreenshotMaker = ExternalScreenshotMaker(
+            screenshotMaker = DocLocScreenshotMaker(
+                screenshotMaker = ExternalScreenshotMaker(
```

Comment:
before changes we priority use `InternalScreenshotMaker` for all screenshot, why we change default screenshots to external for docLocTest?

Reply:
Why we don`t changed that for com/kaspersky/kaspresso/kaspresso/Kaspresso.kt:740?

Reply:
We used External Screenshots only for Doc Loc, for Full window screenshots we'll use internal channel, as for Kaspresso:740 i'll fix it

Reply:
@VladislavSumin  for what do we need to change that in Kaspresso:740 ? That's screenshots only for base tests, and it was originally Internal and fallback for external

Reply:
Ok, I further discussed this questions with @AzamatCherchesov and we came to the conclusion that this decision is justified